Lib/pathlib.py

```diff
@@ -1363,8 +1363,13 @@ def link_to(self, target):
 
     def rename(self, target):
         """
-        Rename this path to the given path,
-        and return a new Path instance pointing to the given path.
+        Rename / move this path to the target path.
```

Comment:
```suggestion
        Rename this path to the target path.
```
I think it's confusing to have two verbs here, maybe we should only keep the more appropriate and clarify if needed in the paragraph under the first line.